wallet/how-to/connect/index.md

```diff
@@ -1,166 +1,313 @@
 ---
 description: Connect to MetaMask via EIP-6963.
 sidebar_position: 1
+toc_max_heading_level: 4
 ---
 
 # Connect to MetaMask
 
 You can connect your dapp to users' MetaMask wallets by detecting MetaMask in their browsers and
 connecting to their accounts.
 
-This page describes how to connect to MetaMask using the wallet detection mechanism introduced by
-[EIP-6963](https://eips.ethereum.org/EIPS/eip-6963).
-EIP-6963 enables [wallet interoperability](../../concepts/wallet-interoperability.md), and shifts
-dapps from relying solely on [`window.ethereum`](detect-metamask.md) for wallet detection.
-If a user has multiple wallet browser extensions installed, you can detect multiple wallets and
-connect to each one without conflict.
+The recommended method for connecting to MetaMask or other browser wallets is using the wallet
+detection mechanism introduced by [EIP-6963](https://eips.ethereum.org/EIPS/eip-6963).
+This method allows you to detect multiple wallets and connect to them without conflicts.
 
-You can connect to MetaMask [using third-party libraries](#use-third-party-libraries) or
-[directly using the Wallet API](#connect-to-metamask-directly).
+:::info
+Learn more about EIP-6963 in [Wallet interoperability](../../concepts/wallet-interoperability.md).
+:::
+
+You can connect to MetaMask [using third-party libraries](#connect-to-metamask-using-third-party-libraries)
+or [directly using Vite](#connect-to-metamask-directly-using-vite).
 
-## Use third-party libraries
+## Connect to MetaMask using third-party libraries
 
 You can connect to MetaMask using the following third-party libraries that support EIP-6963:
 
 - [Wagmi 2+](https://wagmi.sh)
 - [Web3Modal 3+](https://docs.walletconnect.com/web3modal/about)
 - [MIPD Store](https://github.com/wevm/mipd)
-- [RainbowKit](https://www.rainbowkit.com/)
+- [RainbowKit](https://www.rainbowkit.com)
+- [Web3-Onboard](https://onboard.blocknative.com)
+
+## Connect to MetaMask directly using Vite
+
+To connect to MetaMask directly, we recommend implementing support for EIP-6963 using the
+[Vite](https://vitejs.dev/) build tool with [Vanilla TypeScript](#vanilla-typescript) or
```

Comment:
I think "Vanilla Typescript" is not a proper noun, but installed "vanilla" is an adjective to describe the Typescript. I think maybe we should format instances as "vanilla Typescript".